istio-installation.yaml

```diff
@@ -6,11 +6,12 @@ metadata:
 spec:
   istio:
     global:
+      multitenant: false
```

Comment:
I think we should do a multitenant install, because single-tenancy is going away, IIUC.

Reply:
Julie linked to the [list of issues](https://docs.openshift.com/container-platform/3.11/servicemesh-release-notes/servicemesh-release-notes.html#known-issues), one of them is [MAISTRA-453](https://issues.jboss.org/browse/MAISTRA-453).
I can do some clean up with the update to Maistra 0.13.0, assign me the issue once Maistra 0.13.0 is out, (such as remove a Jaeger version tag, and switch to the multi tenant mode), does it sound reasonable ?

Reply:
Whoah, that's a pretty serious issue. OK, that makes sense.